File: src/kairos.app/src/services/time-holiday-entry/mutations/create-time-holiday-entries.ts
export const createTimeHolidayEntriesMutation = `
    mutation ($timeHolidayEntries: [TimeHolidayEntryModel!]!) {
        createTimeHolidayEntries(timeHolidayEntries: $timeHolidayEntries) {
            ids
        }
    }  
`;

File: src/Kairos.Application/TimeHolidayEntry/Commands/CreateTimeHolidayEntries.cs
using System;
using System.Collections.Generic;
using System.Collections.Immutable;
using Kairos.Application.TimeHolidayEntry.Dtos;
using Kairos.Common;

namespace Kairos.Application.TimeHolidayEntry.Commands
{
    public class CreateTimeHolidayEntries : Command<ImmutableList<Guid>>
    {
        public CreateTimeHolidayEntries(params TimeHolidayEntryModel[] timeHolidayEntries)
        {
            TimeHolidayEntries = timeHolidayEntries;
        }

        public IEnumerable<TimeHolidayEntryModel> TimeHolidayEntries { get; }
    }
}